index.html

```diff
@@ -0,0 +1,107 @@
+      and statements. Core JavaScript can be extended for a variety of purposes by supplementing it with additional objects; 
+      for example:
+    </p>
+    <ul>
+      <li>
+       Client-side JavaScript extends the core language by supplying objects to control a browser and its Document Object Model (DOM).
+       For example, client-side extensions allow an application to place elements on an HTML form and respond to user events such as mouse clicks,
+       form input, and page navigation.
+      </li>
+      <li>
+        Server-side JavaScript extends the core language by supplying objects relevant to running JavaScript on a server.
+         For example, server-side extensions allow an application to communicate with a database, provide continuity of information from one invocation to another of the application,
+         or perform file manipulations on a server.
+      </li>
+    </ul>
+    <h2 id="section2">What you should already know</h2>
+    <p>This guide assumes you have the following basic background.</p>
+    <ul>
+      <li>A general understanding of the Internet and the World Wide Web (WWW).</li>
+      <li>Good working knowledge of HyperText Markup Language (HTML).</li>
+      <li>Some programming experience. If you are new to programming, 
+        try one of the tutorials linked on the main page about JavaScript.
+      </li>
+    </ul>
+    <h2 id="section3">JavaScript and Java</h2>
+    <p>
+      JavaScript and Java are similar in some ways but fundamentally different in some others. 
+      The JavaScript language resembles Java but does not have Java's static typing and strong type checking. JavaScript follows most Java expression syntax,
+      naming conventions and basic control-flow constructs which was the reason why it was renamed from LiveScript to JavaScript.
+    </p>
+    <p>
+      In contrast to Java's compile-time system of classes built by declarations,
+      JavaScript supports a runtime system based on a small number of data types representing numeric,
+      Boolean, and string values. JavaScript has a prototype-based object model instead of the more common class-based object model. 
+      The prototype-based model provides dynamic inheritance; 
+      that is, what is inherited can vary for individual objects.
+      JavaScript also supports functions without any special declarative requirements. 
+      Functions can be properties of objects, executing as loosely typed methods.
+    </p>
+    <p>
+      JavaScript is a very free-form language compared to Java.
+      You do not have to declare all variables, classes, and methods.
+      You do not have to be concerned with whether methods are public, private, or protected, 
+      and you do not have to implement interfaces. Variables, parameters, 
+      and function return types are not explicitly typed.
+    </p>
+    </div>
+  </main>
+
+  <aside>
+    <h2 class="main-heading">JS Documentation</h2>
```

Comment:
Please do not add an empty line or empty space in your code without any reason. This is not a good practice accordint to HTML and CSS best practice rules.